vembrane/globals.py

```diff
@@ -102,7 +102,7 @@
     "__package__": None,
 }
 
-globals_whitelist = {
+globals_allowlist = {
```

Comment:
Since this isn't using an "established compound word" anymore, this would be a good opportunity to use more generic (arguably more readable) snake_case, i.e., `globals_allow_list`.
Or even better: `allowed_global_symbols` or just `allowed_globals` since we don't need the `_list` part.

Reply:
yup!